news/_posts/core-weekly/2019-07-31-coreweekly-week-28-2019.md

```diff
@@ -0,0 +1,135 @@
+* [#14552](https://github.com/PrestaShop/PrestaShop/pull/14552): Use variable instead of recomputing, by [@jolelievre](https://github.com/jolelievre)
+* [#14560](https://github.com/PrestaShop/PrestaShop/pull/14560): Typo fix on bug issue template, by [@PierreRambaud](https://github.com/PierreRambaud)
+* [#14576](https://github.com/PrestaShop/PrestaShop/pull/14576): Merge 1.7.6.x to develop - 10/07/2019, by [@matks](https://github.com/matks)
+* [#14592](https://github.com/PrestaShop/PrestaShop/pull/14592): Merge 1.7.6.x to develop - 11/07/2019, by [@matks](https://github.com/matks)
+
+
+### Back office
+
+* [#12997](https://github.com/PrestaShop/PrestaShop/pull/12997): [+] BO: add disable visual effect on translations. Thank you [@francoispeyret](https://github.com/francoispeyret)
+* [#13529](https://github.com/PrestaShop/PrestaShop/pull/13529): Migrate Sell > Catalog > Monitoring listing action. Thank you [@zuk3975](https://github.com/zuk3975)
+* [#14198](https://github.com/PrestaShop/PrestaShop/pull/14198): Add Commands/Queries Profiler tab. Thank you [@sarjon](https://github.com/sarjon)
+* [#14249](https://github.com/PrestaShop/PrestaShop/pull/14249): Case-insensitive hook dispatching. Thank you [@tomas862](https://github.com/tomas862)
+* [#14308](https://github.com/PrestaShop/PrestaShop/pull/14308): #14302 : Duplicate Product in BO with several taxes (multishop). Thank you [@202-ecommerce](https://github.com/202-ecommerce)
+* [#14320](https://github.com/PrestaShop/PrestaShop/pull/14320): Introduces NumberMinMaxFilterType and IntegerMinMaxFilterType. Thank you [@tomas862](https://github.com/tomas862)
+* [#14525](https://github.com/PrestaShop/PrestaShop/pull/14525): Fix php warning in php 7.2 in case of JSON output. Thank you [@202-ecommerce](https://github.com/202-ecommerce)
+* [#14530](https://github.com/PrestaShop/PrestaShop/pull/14530): Use Preview link without  option for seo preview, by [@PierreRambaud](https://github.com/PierreRambaud)
+
+
+### Front office
+
+* [#14437](https://github.com/PrestaShop/PrestaShop/pull/14437): Translate aria-label in modal close button. Thank you [@davidglezz](https://github.com/davidglezz)
+* [#14507](https://github.com/PrestaShop/PrestaShop/pull/14507): Fix bug of FO translations with multiple resources and tpl inclusions. Thank you [@202-ecommerce](https://github.com/202-ecommerce)
+* [#14567](https://github.com/PrestaShop/PrestaShop/pull/14567): Remove image in about us cms page, by [@marionf](https://github.com/marionf)
+
+
+### Tests
+
+* [#14508](https://github.com/PrestaShop/PrestaShop/pull/14508): Integrate Puppeteer to check links on FO and BO, by [@mbadrani](https://github.com/mbadrani)
+* [#14531](https://github.com/PrestaShop/PrestaShop/pull/14531): Remove another faulty e2e test from Travis, by [@matks](https://github.com/matks)
+
+
+## Code changes in the "1.7.6.x" branch (for v1.7.6.0)
```

Comment:
`## Code changes in the "1.7.6.x" branch (for v1.7.6.0)`
=>
`## Code changes in the "1.7.6.x" branch (for v1.7.6.1)` 😉@ttoine